CONTRIBUTING.md

```diff
@@ -1,4 +1,92 @@
-# Contributing
+# Contributing to LumApps Extensions Templates
+
+<h2>Table of contents</h2>
```

Comment:
## ?

Reply:
With ## the `Table of contents` is added to the table of content, and I don't want this